firestore/go.mod

```diff
@@ -3,7 +3,7 @@ module cloud.google.com/go/firestore
 go 1.21
 
 require (
-	cloud.google.com/go v0.116.0
+	cloud.google.com/go v0.117.0
```

Comment:
Why is a test fix PR also updating the version requirement of a dependency for customers?

Reply:
to get Fatalf in testutil.Retry